cli/cmd/encore/test.go

```diff
@@ -50,7 +56,7 @@ func runTests(appRoot, testDir string, args []string) {
 	if err != nil {
 		fatal(err)
 	}
-	os.Exit(streamCommandOutput(stream, true))
+	os.Exit(streamCommandOutput(stream, convertJSON))
```

Comment:
Note before merging, I want to get this working so we still convert the JSON logs to into pretty printed logs within the JSON test format. Hence why this PR is draft still